### PR TITLE
Hotfix/light state error

### PR DESCRIPTION
**Pull request by bcb2000**
Logic for identifying a light capability in a fan, or the fan itself, was not working correctly.  Also, strengthened the checking for keys in the state logic, to handle this scenario better.

### CODE REVIEW

File: custom_components/bond/bond.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import requests
+
+BOND_DEVICE_TYPE_CEILING_FAN = "CF"
+BOND_DEVICE_TYPE_FIREPLACE = "FP"
+BOND_DEVICE_TYPE_MOTORIZED_SHADES = "MS"
+BOND_DEVICE_TYPE_GENERIC_DEVICE = "GX"
```
Comment (marciogranzotto) [inline custom_components/bond/bond.py:6]: if you want to support all device types:
```
CEILING_FAN = "CF"
FIREPLACE = "FP"
AIR_CONDITIONER = "AC"
GARAGE_DOOR = "GD"
SHADES = "MS"
BIDET = "BD"
GENERIC = "GX"
```

File: custom_components/bond/bond.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+BOND_DEVICE_ACTION_PRESET = "Preset"
+BOND_DEVICE_ACTION_TOGGLEOPEN = "ToggleOpen"
+
+# Relating to Ceiling Fan (CF)
+BOND_DEVICE_ACTION_SETSPEED = "SetSpeed"
+BOND_DEVICE_ACTION_INCREASESPEED = "IncreaseSpeed"
+BOND_DEVICE_ACTION_DECREASESPEED = "DecreaseSpeed"
+BOND_DEVICE_ACTION_TURNLIGHTON = "TurnLightOn"
+BOND_DEVICE_ACTION_TURNLIGHTOFF = "TurnLightOff"
+BOND_DEVICE_ACTION_TOGGLELIGHT = "ToggleLight"
```
Comment (marciogranzotto) [inline custom_components/bond/bond.py:31]: Some ceiling fans have Up and Down lights, for that you need:
```
"TurnUpLightOn"
"TurnDownLightOn"
"TurnUpLightOff"
"TurnDownLightOff"
"ToggleUpLight"
"ToggleDownLight"
```

File: custom_components/bond/light.py [hunk 3]
```diff
@@ -1,9 +1,17 @@
 """Bond Home Light Integration"""
 from homeassistant.components.light import (
-    ATTR_BRIGHTNESS, PLATFORM_SCHEMA, Light)
+    ATTR_BRIGHTNESS,PLATFORM_SCHEMA,Light)
```
Comment (marciogranzotto) [inline custom_components/bond/light.py:3]: You can also check if the device supports the actions
```
SetBrightness
SetUpLightBrightness
SetDownLightBrightness
```
and then add this feature to the light

**Review by marciogranzotto — APPROVED** [review]

**Comment by nguyer** [comment]
Closed, due to https://github.com/nguyer/homeassistant-bond-home/pull/11 superseding this PR

**Pull request closed without merging**